src/lib/cache/subCache.cpp

```diff
@@ -735,7 +737,21 @@ void subCacheItemInsert
 
   if (stringFilterP != NULL)
   {
-    cSubP->expression.stringFilter = *stringFilterP;  // Object copy
+    std::string  errorString;
+
+    if (!cSubP->expression.stringFilter.fill(stringFilterP, &errorString))
+    {
+      LM_E(("Runtime Error (%s)", errorString.c_str()));
```

Comment:
Looking to fill() code, it seems that LM_E is generated inside that function. Thus, this LM_E should be removed, to avoid over-logging.